packages/apps-config/src/settings/endpoints.ts

```diff
@@ -247,6 +247,11 @@ function createTestNetworks (t: TFunction): LinkOption[] {
       info: 'westend',
       text: t<string>('rpc.westend', 'Westend (Polkadot Testnet, hosted by Parity)', { ns: 'apps-config' }),
       value: 'wss://westend-rpc.polkadot.io'
+    },
+    {
```

Comment:
Move this up to be alphabetical.

Reply:
I added equilibrium below the dusty. it is ok?